accounting/invoice/templates/default/default.html

```diff
@@ -0,0 +1,134 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link rel="stylesheet" type="text/css" href="{% static 'vendor/css/bootstrap.min.css' %}"/>
+    <link rel="stylesheet" type="text/css" href="{% static 'default/css/default.css' %}"/>
+  </head>
+  <body>
+    <div class="container">
+      <div class="invoice">
+        <header>
+          <div class="row">
+            <div class="col-xs-6">
+              <h1><strong>{{ invoice.provider }}</strong></h1>
+              {% if invoice.provider.occupation %}
+                <h4>{{ invoice.provider.occupation }}</h4>
+              {% endif %}
+            </div>
+            <div id="invoice-metadata" class="col-xs-6">
+              <span id="invoice-number">Invoice Number: {{ invoice.number }}</span><br>
+              <span id="invoice-created">Created: {{ invoice.date|date:"F d, Y" }}</span><br>
+              <span id="invoice-billing-dates">Billed for {{ invoice.billing_start_date|date:"F d, Y" }} to {{ invoice.billing_end_date|date:"F d, Y" }}</span>
+              <span id="invoice-due">Due: {{ invoice.due_date|date:"F d, Y" }}</span>
+            </div>
+          </div>
+          <div class="row">
+            {% for user_type, user in users %}
+              <div class="col-xs-6">
+                <div id="{{ user_type }}-details">
+                  <h5 class="user-header"><strong>{{ user_type|upper }}</strong></h5>
+                  <span class="user-name">{{ user }}</span><br>
+                  <span class="user-address-line-1">{{ user.address.address_line1 }}</span><br>
+                  {% if user.address.address_line2 %}
+                    <span class="user-address-line-2">{{ user.address.address_line2 }}</span><br>
+                  {% endif %}
+                  <span class="user-city">{{ user.address.city }}</span>,
+                  {% if user.address.state %}
+                    <span class="user-state">{{ user.address.state }}</span>,
+                  {% endif %}
+                  <span class="user-zipcode">{{ user.address.zipcode }}</span><br>
+                  <span class="user-country">{{ user.address.country.name }}</span>
+                  {% if user.vat %}
+                    <br><span class="user-vat">VAT number: {{ user.vat }}</span>
+                  {% endif %}
+                </div>
+              </div>
+            {% endfor %}
+          </div>
+          <div class="row">
+            <div id="client-bank-account-details" class="col-xs-6">
+              <h5 class="client-bank-account-header"><strong>BANK ACCOUNT</strong></h5>
+              Bank Name: {{ bank_account.bank.name }}<br>
+              Bank Account Type: {{ bank_account.type|title }}<br>
+              Bank Account Currency: {{ bank_account.currency }}<br>
+              {% for name, value in bank_account_details %}
+                {{ name }}: {{ value }}<br>
+              {% endfor %}
+            </div>
```

Comment:
As mentioned above, to remove to avoid having to maintain two separate and nearly identical templates

Reply:
:+1: but I'm still wondering from my comment above if some people require this for themselves (not whether OpenCraft requires it). I'll have to ask, but at this point I'm feeling worried bugging people *again* about their invoices :P

Reply:
NB: Addressed in another comment